### PR TITLE
add inkview-eg crate, versioned bindings, justfile, examples

### LINKED ISSUE #2: document setup and process to generate bindings
I am trying to add features to select between SDK v.5.19 and v.6.5 but I am struggling a bit with regenerating the bindings. Can you add some documentation about your setup and process of generating the bindings?

Edit: I think I figured it out. I'll send a PR soon.

### PR DESCRIPTION
**Pull request by flxzt**
This adds a couple of things:

- a inkview-eg subcrate that implements [embedded-graphics-core::DrawTarget](https://docs.rs/embedded-graphics-core/latest/embedded_graphics_core/draw_target/trait.DrawTarget.html) to be able to draw to the display with [embedded-graphics](https://crates.io/crates/embedded-graphics) crates. An example is provided featuring something similar to the example in the embedded-graphics docs.
- Adds features to pick SDK versions `5.19` and `6.5` to be able to build for different firmware versions (my device Touch Lux 3 has firmware v5, I believe the Era has v6). The bindings were regenerated wih the latest bindgen and the "official" SDK found [here](https://github.com/pocketbook/SDK_6.3.0). I took out the bindings to `inkinternal.h` and only used `inkview.h` for now. I assume that you only played around with that to try to fix the offset bug?
- Added a minimal example for the bindings crate, mirroring this one: [demo01](https://github.com/pmartin/pocketbook-demo/blob/master/demo01/demo01.cpp)
- add a justfile to install prerequisites, build apps and examples more easily, regenerate bindings, transfer built binaries to the device

Some other notes:

I still am experiencing this offset bug, but I also have it when running for example this [demo](https://github.com/pmartin/pocketbook-demo/blob/master/demo01/demo01.cpp). So I don't think it's an issue with the rust bindings. I'd like to look into it, that would be a follow-up.

fixes #2

### CODE REVIEW

File: inkview/Cargo.toml [hunk 1]
```diff
@@ -7,3 +7,8 @@ edition = "2021"
 
 [dependencies]
 libloading = "0.8.1"
+
+[features]
+default = []
+sdk-5-19 = []
+sdk-6-5 = []
```
Comment (simmsb) [inline inkview/Cargo.toml]:
```suggestion
sdk-5-19 = ["_sdk_selected"]
sdk-6-5 = ["_sdk_selected"]

_sdk_selected = []
```

File: inkview/src/lib.rs [hunk 2]
```diff
@@ -1,17 +1,14 @@
-#[allow(nonstandard_style, unused)]
-#[allow(rustdoc::invalid_html_tags, rustdoc::broken_intra_doc_links)]
 pub mod bindings;
```
Comment (simmsb) [inline inkview/src/lib.rs:4]:
```suggestion
#[cfg(not(feature = "_sdk_selected"))]
compile_error!("No SDK selected, enable one of the 'sdk-*' features.");

pub mod bindings;
```

**Review by simmsb — APPROVED** [review]
Nice, thanks for your work on this!

My only request is that we add something so that users see a helpful error when they try to build the library without one of the SDK features enabled.

**Comment by flxzt** [comment]
That is a good idea. I applied your suggestions

**Pull request merged**